includes/handlers/class-webmention-handler-base.php

```diff
@@ -73,4 +73,14 @@ public function set_webmention_item( $webmention_item ) {
 	 * @return WP_Error|true Return error or true if successful.
 	 */
 	abstract public function parse( Webmention_Request $request );
+
+	/**
+	 * Returns a property from the raw data in the webmention_item.
+	 * Each handler will implement this to transform raw data appropriately.
+	 *
+	 * @param $string $key Property Key. If not provided all properties returned.
+	 *
+	 * @return mixed Return property or false if not found.
+	 */
+	abstract public function get_raw( $key = null );
```

Comment:
It is always the same function, so it should be part of the base handler.

Even further, the raw value is part of the `Webmention_Item` so I would recommend to move it to the Item instead!

Reply:
The mf2 version is different, and eventually, I'd like the json-type version to also be a bit different.

Reply:
But for now they are the same, so we should start generic and if one needs to be different, it can be overwritten in that class.